core/transport/src/main/proto/proto/journal/journal.proto

```diff
@@ -66,6 +67,7 @@ message JournalEntry {
   optional UpdateInodeEntry update_inode = 35;
   optional UpdateInodeDirectoryEntry update_inode_directory = 36;
   optional UpdateInodeFileEntry update_inode_file = 37;
+  optional LoadJobEntry load_job = 53;
```

Comment:
I'm confused about these id number, why do we use 53 after 37?

Reply:
The comment says next id is 53. I don't think these are in order, and maybe some were deleted before